packages/table-dev-app/src/index.tsx

```diff
@@ -24,10 +24,10 @@ import { Nav } from "./nav";
 
 ReactDOM.render(<Nav selected="index" />, document.getElementById("nav"));
 ReactDOM.render(
-    <HotkeysProvider>
-        <OverlaysProvider>
+    <OverlaysProvider>
+        <HotkeysProvider>
```

Comment:
need to add docs about the ordering of these providers. will do that in a separate PR which adds `<BlueprintProvider>`